lib/eblox/data/post.ex

```diff
@@ -6,12 +6,11 @@ defmodule Eblox.Data.Post do
   require Logger
 
   @fsm """
-  idle --> |read| read
-  idle --> |delete| deleted
-  read --> |parse| parsed
-  read --> |parse| errored
-  read --> |delete| deleted
+  idle --> |read!| read
```

Comment:
Auto transition, which does not require an explicit call: https://github.com/am-kantox/finitomata/#special-events